client/transmitter/osc.go

```diff
@@ -327,6 +327,21 @@ func (oe OSCTransmitter) ScoreToOSCBundle(
 
 		switch event := event.(type) {
 		case model.NoteEvent:
+
+			// Add offset for any rests done after the last Note
+			// Additional rests at the end of a code sequence create a timing mismatch
+			// between Part.CurrentOffset and this note event's Offset + duration
+			// Part.CurrentOffset should equal Note Offset + Note Duration, so this statement
+			// Increments Note Duration so that they become equal
+			restOffset := int32(0)
+			if index == len(events)-1 {
+				lastEvent := event
+				// Last event offset + duration must equal score.duration
+				currNoteEnd := int32(math.Round(lastEvent.Duration)) + int32(lastEvent.EventOffset())
+				partOffset := int32(score.CurrentParts[len(score.CurrentParts)-1].CurrentOffset)
```

Comment:
I think selecting the last Part is potentially error-prone, as it might not be the Part that the note belongs to. I think `lastEvent.Part` is probably what we want here.